templates/channel.pug

```diff
@@ -63,7 +63,10 @@ html(lang="en")
               button#newpollbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-poll")
               button#emotelistbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-emote-dead")
               button#chatoptsbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-preferences")
-              button#coolpointsbtn.btn.btn-sm.btn-default(onclick="javascript:showCoolPointsUserPrompt()"): span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-coin-stencil") CoolPoints
+              button#coolpointsbtn.btn.btn-sm.btn-default(onclick="javascript:showCoolPointsUserPrompt()"): span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-coin-stencil")
+                svg.ch-icon.ch-button-icon.ch-cp: use(xlink:href="#ch-icon-ui-coin")
```

Comment:
question: are there two SVGs being rendered here?

Reply:
I think so actually, lol.

Reply:
i think the old "colored" icon is in there. i removed it and it seemed ok?

Reply:
> i think the old "colored" icon is in there. i removed it and it seemed ok?

Yep it was fine, I didn't even realize it was there since it seemed to be either under the stencil or just pushed somewhere else. I've got an idea to utilize the color icon still without needing an extra element.